### PR TITLE
Giving a more structured look to the documentation

### PR DESCRIPTION
**Pull request by PRUBHTEJ**
**Description**
By giving a more structured look to the documentation , it may be possible for the new contributors to easily understand and contribute.

**Tasks**
Add the list of tasks of this PR.
  - [x] Gives the Readme.md a structured look .
  - [x] Makes the Readme.md easy to understand.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,19 +1,21 @@
-# webots-doc
+# Webots-Documentation
```
Comment (omichel) [inline docs/README.md:1]:
```suggestion
# Webots Documentation
```

File: docs/README.md [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
 - https://www.cyberbotics.com/doc/automobile/index
 
-You are very welcome to contribute to make this documentation better!
-In order to proceed, simply fork this repository, make your modifications and
-open a pull request that we will review and merge.
+You are very welcome to contribute to make this documentation better.<p>
+In order to proceed, follow these steps:
+1.Simply fork this repository <p>
+2.Make your modifications &<p>
+3.Open a pull request that we will review and merge.<p>
```
Comment (omichel) [inline docs/README.md:16]: I would add a space after the dots here.

File: docs/README.md [hunk 3]
```diff
@@ -1,19 +1,21 @@
...
 
-## To view a specific version
+## So as to view a specific version:
```
Comment (omichel) [inline docs/README.md:18]: I believe that's too verbose. We should keep title simple (and short).

File: docs/README.md [hunk 4]
```diff
@@ -37,44 +39,44 @@ Or to a github branch (e.g. `reference_proto`) of any public github repository (
 https://www.cyberbotics.com/doc/guide/index?version=remidhum:reference_proto
 ```
 
-## Run the doc locally
+## So as to Run the doc locally,follow these steps:
```
Comment (omichel) [inline docs/README.md:42]: Same comment here.

File: docs/README.md [hunk 5]
```diff
@@ -37,44 +39,44 @@ Or to a github branch (e.g. `reference_proto`) of any public github repository (
...
 
-Set the terminal to the 'docs' directory.
+1.Set the terminal to the 'docs' directory.
```
Comment (omichel) [inline docs/README.md:44]: A space should be added after the dot.

File: docs/README.md (hunk 2 above)
Comment (PRUBHTEJ) [reply docs/README.md:16]: Thanks for the corrections.

File: docs/README.md (hunk 3 above)
Comment (PRUBHTEJ) [reply docs/README.md:18]: Sure, that would be good to.

File: docs/README.md (hunk 5 above)
Comment (omichel) [reply docs/README.md:44]: Still missing this one.

Comment (PRUBHTEJ) [reply docs/README.md:44]: I think I've added a space after the dot but I'll check it again.
Thanks for the review .

Comment (PRUBHTEJ) [reply docs/README.md]: I've added the spaces after the dot, thanks for identifying this .

File: docs/README.md [hunk 6]
```diff
@@ -37,44 +39,44 @@ Or to a github branch (e.g. `reference_proto`) of any public github repository (
...
 - [http://localhost:8000/?url=&book=reference](http://localhost:8000/?url=&book=reference)
 - [http://localhost:8000/?url=&book=blog](http://localhost:8000/?url=&book=blog)
 - [http://localhost:8000/?url=&book=automobile](http://localhost:8000/?url=&book=automobile)
 
 
-## Run the unit tests
+## For running the unit tests, the following steps should be followed:
 
-Install the pep8 module:
+1.Install the pep8 module:
```
Comment (omichel) [inline docs/README.md:72]: Space is missing here as well.

Comment (PRUBHTEJ) [reply docs/README.md]: I have tried to add the spaces over here as well . Sorry for the trouble.

File: docs/README.md [hunk 7]
```diff
@@ -37,44 +39,44 @@ Or to a github branch (e.g. `reference_proto`) of any public github repository (
 https://www.cyberbotics.com/doc/guide/index?version=remidhum:reference_proto
 ```
 
-## Run the doc locally
+## To Run the doc locally,follow these steps:
```
Comment (omichel) [inline docs/README.md:42]: In English a space is added after a coma sign.

File: docs/README.md [hunk 8]
```diff
@@ -1,19 +1,21 @@
...
 - https://www.cyberbotics.com/doc/automobile/index
 
-You are very welcome to contribute to make this documentation better!
-In order to proceed, simply fork this repository, make your modifications and
-open a pull request that we will review and merge.
+You are very welcome to contribute to make this documentation better.<p>
+In order to proceed, follow these steps:
+1. Simply fork this repository <p>
+2. Make your modifications &<p>
+3. Open a pull request that we will review and merge.<p>
```
Comment (omichel) [inline docs/README.md:16]: We should avoid `<p>` in markdown as it is usually not needed.

File: docs/README.md (hunk 7 above)
Comment (PRUBHTEJ) [reply docs/README.md:42]: Thanks for pointing that out . I'll correct that immediately

File: docs/README.md (hunk 8 above)
Comment (PRUBHTEJ) [reply docs/README.md:16]: Should <br/> be used instead of <p> then?

Comment (omichel) [reply docs/README.md:16]: If you want to put some vertical space, an empty line should suffice. But it may not be necessary...

Comment (omichel) [reply docs/README.md:16]: Also we should avoid & signs which are not very literal, prefer "and".

Comment (PRUBHTEJ) [reply docs/README.md:16]: Okay, I'll do that and will commit these changes.

Comment (PRUBHTEJ) [reply docs/README.md]: I have tried to suffice the requirements.

File: docs/README.md [hunk 9]
```diff
@@ -1,19 +1,21 @@
...
 - https://www.cyberbotics.com/doc/reference/index
 - https://www.cyberbotics.com/doc/blog/index
 - https://www.cyberbotics.com/doc/automobile/index
 
-You are very welcome to contribute to make this documentation better!
-In order to proceed, simply fork this repository, make your modifications and
-open a pull request that we will review and merge.
+You are very welcome to contribute to make this documentation better.
+In order to proceed, follow these steps:
+1. Simply fork this repository
```
Comment (omichel) [inline docs/README.md:14]:
```suggestion
1. Fork this repository 
```

Comment (PRUBHTEJ) [reply docs/README.md]: Thanks for pointing that out . I have tried to correct this as per the requirement.

**Pull request merged**